ioperandsmemory.h

```diff
@@ -0,0 +1,21 @@
+#ifndef IOPERANDSMEMORY
+#define IOPERANDSMEMORY
+
+#include "polishcalctypes.h"
+
+namespace PolishCalcComponent {
+
+class IOperandsMemory{
+public:
+    virtual value_t getOperand() = 0;
```

Comment:
Looks like method should not change class state. May be const?

Reply:
yes